lib/discordrb/data/emoji.rb

```diff
@@ -8,18 +8,17 @@ class Emoji
     # @return [String] the emoji name
     attr_reader :name
 
-    # @return [Server] the server of this emoji
+    # @return [Server,] the server of this emoji
     attr_reader :server
 
-    # @return [Array<Role>] roles this emoji is active for
+    # @return [Array<Role>, nil] roles this emoji is active for
```

Comment:
Best to document this condition
```suggestion
    # @return [Array<Role>, nil] roles this emoji is active for, or nil if the emoji's server is unknown
```